sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala

```diff
@@ -1453,4 +1453,35 @@ class StatisticsSuite extends StatisticsCollectionTestBase with TestHiveSingleto
       }
     }
   }
+
+  test("Fix StatisticsCollectionTestBase#getDataSize refer to ChecksumFileSystem#isChecksumFile") {
```

Comment:
Please add `SPARK-28518` prefix to the test case name.

Reply:
OK. Done

Reply:
Thanks!